source/class/qx/ui/table/model/Abstract.js

```diff
@@ -66,11 +66,22 @@ qx.Class.define("qx.ui.table.model.Abstract", {
     this.__columnIndexMap = {};
   },
 
+  statics: {
+    /**
+     * Member to control if a table should throw an error when you try to change the
+     * data model data whilst there is an incomplete edit. It could possibly break
+     * current implementations so only introduce the change from QX v8.
+     * Ref: https://github.com/qooxdoo/qooxdoo/pull/10377#discussion_r818697343
+     */
+    THROW_ON_MODEL_CHANGE_DURING_EDIT: qx.core.Environment.get("qx.version") >= "8.0.0"
```

Comment:
Just realized this won't work. As soon as we get to v10, it'll fail because as a string, "10.0.0" < "8.0.0". I do believe there's a way to do this with numbers or proper version number comparision, though. @johnspackman @hkollmann isn't there a way to do it?

Reply:
But maybe this should be done in the `defer()` magic method?

Reply:
Isn't `qx.version` sometimes not a number, e.g., "7.1.0-beta.1" or something? Doesn't it require a semver comparison?

Reply:
`parseInt` stops at the first non-digit, so `parseInt(qx.core.Environment.get("qx.version"), 10)` will be the first number.

Reply:
@johnspackman 's suggestion of using `parseInt` solves the problem in this case. I suggest we go with that.